src/librustc/util/ppaux.rs

```diff
@@ -905,6 +924,15 @@ impl fmt::Debug for ty::FloatVid {
 
 impl fmt::Debug for ty::RegionVid {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        if let Some((region, counter)) = get_highlight_region() {
+            debug!("RegionVid.fmt: region={:?} self={:?} counter={:?}", region, self, counter);
+            return if *self == region {
+                write!(f, "'{:?}", counter)
+            } else {
+                write!(f, "")
```

Comment:
well, I know I sort of asked for this, but I think in a way I liked `'_` better -- in particular, I'd like to see a test that shows what happens when you elaborate a struct type like `TyCtxt` -- I'd like to see `TyCtxt<'_, '1, '_>`, in particular. =)

Reply:
I think instead I would modify the `TyRef` code to look for either `"'_"` or `""` and just skip printing in that case.

Reply:
Fixed this in recent commit.